src/schema/v2/me/conversation/submit_inquiry_request_mutation.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  GraphQLInputObjectType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLUnionType,
+} from "graphql"
+import { GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { InternalIDFields } from "schema/v2/object_identification"
+import { ResolverContext } from "types/graphql"
+import { ArtworkType } from "schema/v2/artwork/index"
+import { ShowType } from "schema/v2/show"
+import { UserType } from "schema/v2/user"
+import { LocationType } from "schema/v2/location"
+
+const InquiryItemType = new GraphQLUnionType({
+  name: "InquiryItemType",
+  types: [ArtworkType, ShowType],
+  resolveType: ({ __typename }) => {
+    switch (__typename) {
+      case "Artwork":
+        return ArtworkType
+      case "PartnerShow":
+        return ShowType
+      default:
+        return null
+    }
+  },
+})
+
+const InquiryQuestionInput = new GraphQLInputObjectType({
+  name: "InquiryQuestionInput",
+  fields: () => ({
+    questionID: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    details: {
+      type: GraphQLString,
+    },
+  }),
+})
+
+export const InquiryRequest = new GraphQLObjectType<any, ResolverContext>({
+  name: "InquiryRequest",
+  description: "A request to inquire on an artwork",
+  fields: () => ({
+    ...InternalIDFields,
+    inquireable: {
+      type: InquiryItemType,
+      resolve: (result) => {
+        const { inquireable_type, inquireable } = result
+        return {
+          __typename: inquireable_type,
+          ...inquireable,
+        }
+      },
+    },
+    inquirer: {
+      type: UserType,
+      resolve: ({ inquirer }, _, { userByIDLoader }) => {
+        const { id } = inquirer
+        return userByIDLoader(id)
+      },
+    },
+    inquiryShippingLocation: {
```

Comment:
small nit: given that this appears in the context of an inquiry request, I think we can omit the `inquiry` prefixes as that's somewhat implied. This could just be `shippingLocation`.